lib/python/treadmill/sproc/garbage_collector.py

```diff
@@ -0,0 +1,57 @@
+"""Garbage collector for LDAP and IPA.
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import logging
+import time
+import click
+from treadmill import plugin_manager
+from treadmill import cli
+
+_LOGGER = logging.getLogger(__name__)
+_MODULE = 'treadmill.garbage_collector'
+
+
+def _run_gc(gc_plugins, once, interval):
```

Comment:
nit - i would keep once/interval/sleep in the main function - bit too much arguments that are barely used.